expression/util.go

```diff
@@ -176,6 +176,26 @@ func ExtractColumnSet(exprs []Expression) *intsets.Sparse {
 	return set
 }
 
+// ExtractColumnSetSimple extracts the different values of `UniqueId` for columns in expressions.
+func ExtractColumnSetSimple(expr Expression) *intsets.Sparse {
```

Comment:
You can change the `ExtractColumnSet`'s signature to `ExtractColumnSet(exprs ...Expression)`.
Then you can easily call it by `ExtractColumnSet(a), ExtractColumnSet(a, b), Extract(exprSlice...)`.

This way we don't need two functions.